.github/workflows/verify.yaml

```diff
@@ -0,0 +1,75 @@
+name: verify
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
+
+      - name: Install ShellCheck and shfmt
+        run: |
+          sudo apt update
+          sudo apt install -y shellcheck
+          go install mvdan.cc/sh/v3/cmd/shfmt@latest
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: latest
+          only-new-issues: true
+          args: --timeout=5m
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I would say these lines are not necessary. The token itself is created by the runner for each job, and a repo this small wouldn't risk timing out (and if it does, then that would more likely be a bug in golangci-lint)

Reply:
I think the job was failing without the env(only for the v6 version)